shared/src/Datadog.Trace.ClrProfiler.Native/single_step_guard_rails.cpp

```diff
@@ -84,6 +88,7 @@ HRESULT SingleStepGuardRails::CheckRuntime(const RuntimeInformation& runtimeInfo
             {
                 // supported
                 Log::Debug("SingleStepGuardRails::CheckRuntime: Supported .NET runtime version detected, continuing with single step instrumentation");
+                SetInjectResult("success", "Successfully configured ddtrace package", "success");
```

Comment:
nit: `ddtrace` isn't a .NET thing 😄 

```suggestion
                SetInjectResult("success", "Successfully configured automatic instrumentation", "success");
```